framework/wazuh/core/indexer/__init__.py

```diff
@@ -129,14 +128,14 @@ async def create_indexer(
 
     Parameters
     ----------
-    host : str
-        Location of the Wazuh Indexer.
+    hosts : List[str]
+        List of hosts of the Wazuh Indexers.
+    ports : List[int]
+        List of ports of the Wazuh Indexers to connect with.
```

Comment:
```suggestion
    hosts : List[str]
        Wazuh indexer nodes hosts.
    ports : List[int]
        Wazuh indexer nodes ports.
```

Reply:
It was changed in edcb389a839c82846356d2ca9e981f422df1c442.